clang/lib/AST/Interp/Descriptor.h

```diff
@@ -109,6 +109,8 @@ struct Descriptor final {
   const bool IsTemporary = false;
   /// Flag indicating if the block is an array.
   const bool IsArray = false;
+  /// Flag indicating if this is a dummy descriptor.
+  const bool IsDummy = false;
```

Comment:
Should we be considering using a bit-field to pack these together at some point? (As an NFC follow-up)

Reply:
Sure, I'll make a note.